docs/Contributing/API-for-contributors.md

```diff
@@ -650,11 +656,23 @@ Content-Type: application/octet-stream
 --------------------------f02md47480und42y
 ```
 
-
 ##### Default response
 
 `Status: 200`
 
+```json
+{
```

Comment:
@marko-lisica wanted to check: should this (and in other responses below) be nested under a key? So like
```json
{
    "token": {
        "id": 1 // and so on...
    }
}
```

I think this is how most other responses are structured.

Reply:
@jahzielv I agree. I just updated PR (nested it under `abm_token`)